README.md

```diff
@@ -1,8 +1,307 @@
+  propagate             Propagate operation in the gateway actor
+  whitelist-propagator  Whitelist propagators in the gateway actor
+  send-value            Send value to an address within a subnet
+  wallet-new            Create new wallet in subnet
+  help                  Print this message or the help of the given subcommand(s)
+
+Options:
+  -c, --config-path <CONFIG_PATH>  The toml config file path for IPC Agent, default to ${HOME}/.ipc_agent/config.toml
+  -h, --help                       Print help
+  -V, --version                    Print version
+```
+
+## Infrastructure
+IPC currently uses [a fork of Lotus](https://github.com/consensus-shipyard/lotus), that we like to call _Eudico_, to run its subnets. The IPC agent does nothing by itself, and is just an orchestrator over existing subnet deployments. To ease the deployment of new subnets and nodes, we provide a set of convenient scripts to deploy all the infrastructure required to run IPC. 
+
+### Install infrastructure scripts
+[Eudico](https://github.com/consensus-shipyard/lotus/tree/spacenet/scripts/ipc) provides a set of infrastructure scripts, which assume a working installation of Docker. To install Docker [follow this link])(https://docs.docker.com/get-docker/) and choose your working environment.
+
+> Some users have reported some issues trying to build the required images using Docker Desktop, if this is the case, try installing a version of [Docker engine](https://docs.docker.com/engine/install/#server) supported by your system.
+
+With Docker installed, you can then `make install-infra` in the root of the `ipc-agent`. This make rule will clone the eudico repo, build the docker image that you need to run subnets, and install the infrastructure scripts in the `./bin` folder. In Unix-based systems, it is highly recommended to include your user inside the `docker` group to avoid having to run many of the commands from this tutorial using `sudo`. You can achieve this running the following commands:
+```bash
+$ sudo usermod -aG docker $USER newgrp docker
+$ newgrp docker
+```
+
+To check if the installation of the image and all infra-related scripts was successful you can run the following command, and it should return a similar output: 
+```bash
+$ docker images | grep eudico
+eudico                      latest        8fb6db609712   2 minutes ago   341MB
+```
 
 ## Usage
-- Build
-- Config
+### Configuration
+If you are running the agent for the first time, the first thing you need to do is to create a new config. The default config path for the agent is `~/.ipc_agent/config.toml`. The agent will always try to pick up the config from this path unless told otherwise. To populate a sample config file in the default path, you can run the following command:
+```bash
+$ ./bin/ipc_agent init-config
+```
+If you `cat ~/.ipc_agent/config.toml` you should see a new config populated with a sample root and subnet configurations.
```

Comment:
> @adlrocha on Slack: Be sure to use 0.0.0.0 for json_rpc_address = "0.0.0.0:3030" on your ~/.ipc_agent/config.toml to allow conncetions from external IP addresses (Docker has its own IP prefixes)

Maybe we should add a note here too?

Reply:
Also of note: reload-config does not open a new socket, so this either needs to be done prior to starting or requires relaunching the process.